tests/TestAll.py

```diff
@@ -0,0 +1,37 @@
+from tradingpatterns.hard_data import generate_sample_df_with_pattern
+from tradingpatterns.tradingpatterns import detect_patterns, plot_patterns
+
+def test_detect_patterns():
+    # Generate data with different patterns
+    df_head_shoulder = generate_sample_df_with_pattern("Head and Shoulder")
+    df_inv_shoulder = generate_sample_df_with_pattern("Inverse Head and Shoulder")
+    df_double_top = generate_sample_df_with_pattern("Double Top")
+    df_double_bottom = generate_sample_df_with_pattern("Double Bottom")
+    df_triple_top = generate_sample_df_with_pattern("Triple Top")
+    df_triple_bottom = generate_sample_df_with_pattern("Triple Bottom")
+
+    plot_patterns(df_head_shoulder)
```

Comment:
Do let me know if you are working on improving the plotting function based on detection rather than just plotting the candlesticks